mmf/configs/datasets/visual_genome/masked.yaml

```diff
@@ -0,0 +1,64 @@
+dataset_config:
+  masked_visual_genome:
+      data_dir: ${env.data_dir}
+      depth_first: false
+      fast_read: false
+      use_images: false
+      use_features: true
+      add_answer: true
+      features:
+          train:
+          - visual_genome/detectron_fix_100/fc6/,visual_genome/resnet152/
```

Comment:
VisualGenome is currently automatically downloading using custom downloader. We should make a note of this and migrate it soon as well.